Dockerfile

```diff
@@ -48,89 +48,110 @@ RUN update-alternatives --set gcc /usr/bin/gcc-15 \
+RUN 	cd /tmp/Hyprland/subprojects/hyprland-protocols && \
 	meson setup build && \
 	ninja -C build && \
-	ninja -C build install
+	ninja -C build install && \
+	tar -cvf /tmp/hyprland-protocols_v0.6.4-rec.tar.gz -T build/meson-logs/install-log.txt
 
 # Hyprland Utils
 
-RUN 	git clone https://github.com/hyprwm/hyprlock && \
+RUN 	cd /tmp;git clone https://github.com/hyprwm/hyprlock && \
 	cd hyprlock && git checkout v0.9.0 && \
 	cmake --no-warn-unused-cli -DCMAKE_BUILD_TYPE:STRING=Release -S . -B ./build && \
 	cmake --build ./build --config Release --target hyprlock -j`nproc 2>/dev/null || getconf _NPROCESSORS_CONF` && \
-	cmake --install build
+	cmake --install build && \
+	tar -cvf /tmp/hyprlock_v0.9.0.tar.gz -T build/install_manifest.txt
 
 RUN 	git clone https://github.com/hyprwm/hyprpaper && \
 	cd hyprpaper && git checkout v0.7.5 && \
 	cmake --no-warn-unused-cli -DCMAKE_BUILD_TYPE:STRING=Release -DCMAKE_INSTALL_PREFIX:PATH=/usr -S . -B ./build && \
 	cmake --build ./build --config Release --target hyprpaper -j`nproc 2>/dev/null || getconf _NPROCESSORS_CONF` && \
-	cmake --install ./build
+	cmake --install ./build && \
+	tar -cvf /tmp/hyprpaper_v0.7.5.tar.gz -T build/install_manifest.txt
 
 RUN 	git clone https://github.com/hyprwm/hypridle && \
 	cd hypridle && git checkout v0.1.6 && \
 	cmake --no-warn-unused-cli -DCMAKE_BUILD_TYPE:STRING=Release -S . -B ./build && \
 	cmake --build ./build --config Release --target hypridle -j`nproc 2>/dev/null || getconf _NPROCESSORS_CONF` && \
-	cmake --install build
+	cmake --install build && \
+	tar -cvf /tmp/hypridle_v0.1.6.tar.gz -T build/install_manifest.txt
 
 RUN 	git clone --recursive https://github.com/hyprwm/xdg-desktop-portal-hyprland && \
 	cd xdg-desktop-portal-hyprland && git checkout v1.3.10 && \
 	cmake -DCMAKE_INSTALL_LIBEXECDIR=/usr/lib -DCMAKE_INSTALL_PREFIX=/usr -B build && \
 	cmake --build build && \
-	cmake --install build
-
-RUN git clone https://github.com/hyprwm/hyprland-qtutils && \
-    cd hyprland-qtutils && git checkout v0.1.3 && \
-    cmake --no-warn-unused-cli -DCMAKE_BUILD_TYPE:STRING=Release -DCMAKE_INSTALL_PREFIX:PATH=/usr -S . -B ./build && \
-    cmake --build ./build --config Release --target all -j`nproc 2>/dev/null || getconf NPROCESSORS_CONF` && \
-    cmake --install build
+	cmake --install build && \
+	tar -cvf /tmp/xdg-desktop-portal-hyprland_v1.3.10.tar.gz -T build/install_manifest.txt
+
+RUN 	git clone https://github.com/hyprwm/hyprland-qtutils && \
+    	cd hyprland-qtutils && git checkout v0.1.4 && \
+    	cmake --no-warn-unused-cli -DCMAKE_BUILD_TYPE:STRING=Release -DCMAKE_INSTALL_PREFIX:PATH=/usr -S . -B ./build && \
+    	cmake --build ./build --config Release --target all -j`nproc 2>/dev/null || getconf NPROCESSORS_CONF` && \
+    	cmake --install build && \
+	tar -cvf /tmp/hyprland-qtutils_v0.1.4.tar.gz -T build/install_manifest.txt
+
+RUN 	git clone https://github.com/hyprwm/hyprland-qt-support && \
```

Comment:
Huh weird - I never noticed this repo before. I wonder what's been breaking that I've not been aware of 😄